examples/ssd/caffe2npz.py

```diff
@@ -59,17 +61,18 @@ def add_link(self, name, link):
                 data = data.reshape((-1, 4) + data.shape[1:])
                 data[:, [1, 0, 3, 2]] = data.copy()
             print('{:s} -> {:s} (xy -> yx)'.format(name, new_name))
-        else:
+        elif match:
```

Comment:
Yes, but the reason why I added `match` is because it shows all attributes even if they are not layers (e.g. [self.verbose](https://github.com/Hakuyume/chainer-ssd/blob/master/lib/caffe.py#L45)).